.github/workflows/pr-website.yaml

```diff
@@ -30,7 +30,14 @@ jobs:
           cache: 'pnpm'
 
       - name: Install dependencies
-        run: pnpm install --prefer-offline --frozen-lockfile
+        run: |
+          yarn install
```

Comment:
This one is wrong.
I assume we cannot replace `pnpm` by `yarn` in such a way.
This seems to be what explain the following failed check: https://github.com/janus-idp/backstage-plugins/actions/runs/8618570673/job/23648800386#step:5:13

Reply:
good catch, oops!

why are we using BOTH pnpm and yarn?? Seems strange to need two different package managers (also we only use yarn in showcase and downstream, afaik... why use two package managers here? )